packages/client/src/forms/index.ts

```diff
@@ -641,13 +644,13 @@ export interface IQueryMap {
 }
 export interface ILoaderButton extends IFormFieldBase {
   type: typeof FETCH_BUTTON
-  queryMap: IQueryMap
+  queryMap?: IQueryMap
   queryData?: IQuery
-  querySelectorInput: IFieldInput
+  querySelectorInput?: IFieldInput
   onFetch?: (response: any) => void
-  modalTitle: MessageDescriptor
-  successTitle: MessageDescriptor
-  errorTitle: MessageDescriptor
+  modalTitle?: MessageDescriptor
+  successTitle?: MessageDescriptor
+  errorTitle?: MessageDescriptor
```

Comment:
Why do we need to make all these optional? We aren't event modifying this field?

Reply:
I will revert